src/components/FiltersBar/index.js

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState, useContext } from "react";
```

Comment:
Não está sendo utilizado o useState
```suggestion
import React, { useContext } from "react";
```